tests/test_termui.py

```diff
@@ -446,3 +446,13 @@ def cli(password):
 
     if prompt == "Confirm Password":
         assert "Confirm Password: " in result.output
+
+
+def test_show_default_false_hides_prompt(runner):
+    @click.command()
+    @click.option("--arg1", show_default=False, prompt=True, default="my-default-value")
+    def cmd(arg1):
+        pass
+
+    result = runner.invoke(cmd, input="my-input", standalone_mode=False)
```

Comment:
Maybe add a comment to explain what you are actually checking here.

That might help people in the future that will read your test

Reply:
Done